content/events/2018-detroit/program/tom-kowalski.md

```diff
@@ -0,0 +1,17 @@
++++
+Talk_date = ""
+Talk_start_time = ""
+Talk_end_time = ""
+Title = "Devops In a Serverless World"
+Type = "talk"
```

Comment:
Is there any differentiator b/w presentation and ignite?

Reply:
yep, not sure if you mean in general or this file specifically. But, ignites are rendered differently in the output.